src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -21,29 +21,13 @@ namespace OpenTelemetry.Metrics
     /// </summary>
     public static class MeterProviderBuilderExtensions
     {
-        /// <summary>
-        /// Sets default collection period.
-        /// </summary>
-        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
-        /// <param name="periodMilliseconds">Perion in milliseconds.</param>
-        /// <returns><see cref="MeterProvider"/>.</returns>
-        public static MeterProviderBuilder SetDefaultCollectionPeriod(this MeterProviderBuilder meterProviderBuilder, int periodMilliseconds)
-        {
-            if (meterProviderBuilder is MeterProviderBuilderSdk meterProviderBuilderSdk)
-            {
-                return meterProviderBuilderSdk.SetDefaultCollectionPeriod(periodMilliseconds);
-            }
-
-            return meterProviderBuilder;
-        }
-
         /// <summary>
         /// Add measurement processor.
         /// </summary>
         /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
-        /// <param name="processor">Measurement Processors.</param>
+        /// <param name="processor">Measurement Processor.</param>
         /// <returns><see cref="MeterProvider"/>.</returns>
-        public static MeterProviderBuilder AddProcessor(this MeterProviderBuilder meterProviderBuilder, MeasurementProcessor processor)
+        public static MeterProviderBuilder AddMeasurementProcessor(this MeterProviderBuilder meterProviderBuilder, MeasurementProcessor processor)
```

Comment:
this might be removed from the spec. if that happens, we'll also remove here.

Reply:
If we need it, it might be `SetMeasurementProcessor` since we don't allow multiple processors.